stock_orderpoint_manual_procurement/models/stock_warehouse_orderpoint.py

```diff
@@ -41,6 +41,13 @@ def _get_procure_recommended_qty(self, virtual_qty, op_qtys):
 
     @api.depends("product_min_qty", "product_id", "qty_multiple")
     def _compute_procure_recommended(self):
+        # '_quantity_in_progress' override in 'purchase_stock' method has not
+        # been designed to work with NewIds (resulting in KeyError exceptions).
+        # To circumvent this, we knowingly skip such records here.
+        for op in self:
+            # Set values on the current recordset (mandatory to avoid CacheMiss errors)
+            op.procure_recommended_qty = op.procure_recommended_date = False
+        self = self.browse([o.id for o in self if o.id])
```

Comment:
just do it in the loop w/out changing self and w/out looping twice

Reply:
We can not do this because the issue is triggered in `_quantity_in_progress` called from `self` the line above.